From: Paul Martin <pm@debian.org>
Date: Wed, 30 Aug 2017 16:24:03 +0200
Subject: manpage
Closes: #101272, #335060

Patches from: J S Bygott <jsb_no_spm@jsbygott.fsnet.co.uk>

General fixes to manpages.
---
 logrotate.8.in | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/logrotate.8.in b/logrotate.8.in
index 7f5588c..2b728d1 100644
--- a/logrotate.8.in
+++ b/logrotate.8.in
@@ -634,10 +634,14 @@ left l l.
 .BR strtoul (3),
 <https://github.com/logrotate/logrotate>
 
+.SH NOTES
+The \fBkillall\fR(1) program in Debian is found in the \fIpsmisc\fR package.
+
 .SH AUTHORS
 .nf
 Erik Troan, Preston Brown, Jan Kaluza.
 
 <https://github.com/logrotate/logrotate>
 
+Corrections and changes for Debian by Paul Martin <pm@debian.org>
 .fi
